src/reporting/reporters/post-pipeline/console/index.ts

```diff
@@ -58,5 +59,6 @@ export const consoleReporter: Reporter<
     const endedAt = performance.now() - startedAt;
     printEndPipeline(endedAt);
     printPipelineOutcome(data, status, warnings as Nsci.Warnings);
+    printWarningsDescription();
```

Comment:
As it was mentioned in the issue, I feel like printing that table should be done via another CLI command and not being part of the "post-pipeline" step as it pollutes the global outcome.

You could create a specific CLI command in `bin/cli.ts` and invoke it from there

Reply:
This file has not been moved but this is good enough for now.